homeworks/homework-1/app/src/components/CourseList.js

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import {ListGroup, ListGroupItem, Button, Form, FormControl, FormGroup, Panel} from 'react-bootstrap';
+import CourseListItem from "./CourseListItem";
+import Header from "./Header";
+
+class CourseList extends React.Component {
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      courses: [
+        {
+          id: 1,
+          title: 'Course 1',
+          duration: '180',
+          date: '12.12.12',
+          description: 'Description1'
+        },
+        {
+          id: 2,
+          title: 'Course 2',
+          duration: '20',
+          date: '12.12.12',
+          description: 'Description2'
+        },
+        {
+          id: 3,
+          title: 'Course 3',
+          duration: '100',
+          date: '12.12.12',
+          description: 'Description3'
+        }
+      ],
+      greeting: "Courses"
+    };
+  }
+
+  render() {
+    const {
+      courses
+    } = this.state;
+
+    return (
+      <div>
+        <Header greeting={this.state.greeting}/>
+        <Form inline>
+          <FormGroup controlId="searchParameter">
+            <FormControl type="text" placeholder="enter name or date"/>
+          </FormGroup>{' '}
```

Comment:
Please avoid such kind of magic `{' '}`) It very confusing use styles instead of this. 

P.S. Also sometimes we want to make some magic in styles like `margin-right: 11px`, `11px` it's very unclear number, try to use `10px` or `5px`. The main idea try to avoid magic numbers